src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -10,11 +10,30 @@
 
 namespace Microsoft.ML.Transforms.Text
 {
-
     /// <summary>
-    /// Produces a bag of counts of ngrams (sequences of consecutive words) in a given text.
-    /// It does so by building a dictionary of ngrams and using the id in the dictionary as the index in the bag.
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="ITransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | Vector of [Text](xref:Microsoft.ML.Data.TextDataViewType) |
+    /// | Output column data type | Vector of known-size of <xref:System.Single> |
+    ///
+    /// The resulting <xref:Microsoft.ML.ITransformer> creates a new column, named as specified in the output column name parameters, and
+    /// produces a vector of counts of n-grams (sequences of n consecutive words) from a given data.
```

Comment:
>from a given data [](start = 82, length = 17)

maybe:
counts of n-grams -> counts of occurrences of ngrams 
from a given data -> in the input vector of words

Reply:
@singlis suggested "vector of n-gram counts" which I think is good